src/main/java/com/github/manolo8/darkbot/extensions/features/decorators/InstructionProviderDecorator.java

```diff
@@ -13,7 +14,10 @@ public class InstructionProviderDecorator extends FeatureDecorator<InstructionPr
 
     @Override
     protected void load(FeatureDefinition<InstructionProvider> fd, InstructionProvider obj) {
-        if (obj instanceof Module && !(obj instanceof TemporalModule)) showInstructions(obj, fd.getName());
+        if (obj instanceof Module
+                && !(obj instanceof TemporalModule)
+                && ConfigEntity.INSTANCE.getConfig().BOT_SETTINGS.OTHER.SHOW_INSTRUCTIONS
+        ) showInstructions(obj, fd.getName());
```

Comment:
```suggestion
                && ConfigEntity.INSTANCE.getConfig().BOT_SETTINGS.OTHER.SHOW_INSTRUCTIONS)
                    showInstructions(obj, fd.getName());
```
minor formatting change